autogen.sh

```diff
@@ -11,3 +11,5 @@ autoreconf --verbose --force --install || exit $?
 
 cd "$cwd"
 test -z "$NOCONFIGURE" && "$srcdir/configure" $@
+
+gtkdocize || exit 1
```

Comment:
Sounds good I'd appreciate such a way. It would be nice if the system could look for it. I'll try to make a sketch with a test statement.